lib/openstudio-standards/btap/costing/envelope_costing.rb

```diff
@@ -34,8 +34,12 @@ def cost_audit_envelope(prototype_creator)
 
       # Get SpaceType defined for space. If not defined it will skip the
       # spacetype. May have to deal with Attic spaces.
-      if space.spaceType.empty? or 
-         space.spaceType.get.standardsSpaceType.empty? or 
+      #
+      # rd2: Once BTAP finally switches over to the 'structure'-based
+      #      construction assignment approach, spacetypes will no longer be
+      #      required. Check may no longer be necessary?
+      if space.spaceType.empty? or
```

Comment:
Once BTAP moves away from _spacetype_-specific constructions, I suspect this check may no longer be needed.